src/main/java/com/synopsys/integration/jenkins/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -441,8 +443,30 @@ protected Execution(@Nonnull StepContext context) throws InterruptedException, I
 
         @Override
         protected Integer run() throws PluginExceptionHandler, ScannerException {
-            return ScanCommandsFactory.createPipelineCommand(run, listener, envVars, launcher, node, workspace)
-                .initializeScanner(getParametersMap(workspace, listener));
+            LoggerWrapper logger = new LoggerWrapper(listener);
+            logger.println("**************************** START EXECUTION OF SYNOPSYS SECURITY SCAN ****************************");
+
+            Integer result = null;
+            Exception caughtException = null;
+
+            try {
+                result = Integer.valueOf(ScanParametersFactory
+                        .createPipelineCommand(run, listener, envVars, launcher, node, workspace)
+                        .initializeScanner(getParametersMap(workspace, listener)));
+            } catch (PluginExceptionHandler | ScannerException e) {
+                caughtException = e;
+            } finally {
+                logger.println("**************************** END EXECUTION OF SYNOPSYS SECURITY SCAN ****************************");
+                if (caughtException != null) {
```

Comment:
Is there any specific reason, we are not handling this exception in catch block?

Reply:
Nice catch!
Sorry, forgot to move this segment inside  `catch` block